packages/story-editor/src/components/videoTrim/useVideoTrimMode.js

```diff
@@ -61,7 +61,7 @@ function useVideoTrimMode() {
       return false;
     }
     const { isTranscoding, isMuting, isTrimming, local, isExternal } =
-      selectedElement.resource || {};
+      selectedElement.resource;
```

Comment:
Changed in https://github.com/google/web-stories-wp/pull/9098

Reply:
Oh, but that isn't merged yet, so won't pull that in. But I'll leave this line unchanged instead.

Reply:
Done in 2098a7ba5c.